src/components/Select.vue

```diff
@@ -612,8 +612,7 @@
 
       /**
        * When true, existing options will be filtered
-       * by the search text. Should not be used in conjunction
-       * with taggable.
+       * by the search text.
```

Comment:
I couldn't see any reason why taggable & filterable can't be used together anymore. Let me know if I'm wrong.